cypress/support/pages/dashboard/pageCommonDashboard.js

```diff
@@ -0,0 +1,10 @@
+'use strict'
+
+class Dashboard_Common {
+
+  arriveOnDashboard() {
```

Comment:
We should go the user journey and reach the dashboard by clicking somewhere or automatically after login. 
Is it necessary here to open the dashboard page by using cy.visit?

Reply:
yes, we can adapt it in a follow up (#2039) since it was already on the main before.